File: template.go
package template

import (
	"bufio"
	"fmt"
	"os"
)

func main() {


	fmt.Println(partone("input.txt"))
	fmt.Println(parttwo("input.txt"))
}

func partone(fileName string) (answer int) {

	file, _ := os.Open(fileName)

	scanner := bufio.NewScanner(file)
	var content string

	for scanner.Scan(){
		line := scanner.Text()
		content += line
		answer += 1
	}


	return answer
}
func parttwo(fileName string) (answer int) {

	file, _ := os.Open(fileName)

	scanner := bufio.NewScanner(file)
	var content string

	for scanner.Scan(){
		line := scanner.Text()
		content += line
		answer += 1
	}


	return answer
}

File: template.py
import os
file_path = os.path.dirname(os.path.realpath(__file__)) + "/input.txt"
file_input = open(file_path, "r").read().strip()
file_lines = file_input.split("\n")
answer = 0

for line in file_lines:
  
  answer += 1

print(answer)